nightguard WatchKit App/ExtensionDelegate.swift

```diff
@@ -112,6 +112,13 @@ class ExtensionDelegate: NSObject, WKApplicationDelegate {
     func applicationDidBecomeActive() {
         // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
         AppState.isUIActive = true
+//        NotificationCenter.default.post(name: .refreshDataOnAppBecameActive, object: nil)
+        //let's try also applicationWillEnterForeground()
```

Comment:
This can be removed, right?

Reply:
Well, yes, I left it in, because I am not sure which version is better. I did not really see any difference during the TestFlights. But I could imagine that applicationWillEnterForeground is a bit faster than applicationDidBecomeActive? What do you think?

Reply:
Yes - it should happen before.